allay/plugin.py

```diff
@@ -0,0 +1,29 @@
+from beet import Context
+from beet.core.container import Drop
+from beet.core.file import TextFile
+from beet.contrib.messages import Message
+from beet.library.base import NamespaceFile
+from beet.library.data_pack import DataPack
+
+from tokenstream import InvalidSyntax
+
+from functools import cached_property
+
+from .parser import Parser
+
+class AllayMessage(TextFile, NamespaceFile):
+    scope = ("messages",)
```

Comment:
Does this need to be a tuple instead of a list?

Reply:
yes, it's generally the format for plugins